### ISSUE TITLE
Phing Coding Standard check fails on phpstan run

### ISSUE THREAD

**Issue by TomasVotruba** [shopsys/shopsys#382]
I installed repo with:

```bash
git clone git@github.com:shopsys/shopsys.git
cd shopsys
composer update
```

I found [coding standard](https://github.com/shopsys/shopsys/blob/master/docs/contributing/coding-standards.md#coding-standards) docs (I wish it was findable in `README.md` with "Ctrl + F" as in other packages, it took me 3 minutes to find it).

There is this command:

```bash
php phing standards
```

I run it and it fails on PHPStan not finding a class:

```bash
Shopsys Framework > phpstan:

    0/1940 [░░░░░░░░░░░░░░░░░░░░░░░░░░░░]   0%PHP Fatal error:  Trait 'Tests\ShopBundle\Test\Codeception\_generated\AcceptanceTesterActions' not found in /var/www/shopsys/project-base/tests/ShopBundle/Test/Codeception/AcceptanceTester.php on line 28

BUILD FAILED
/var/www/shopsys/build.xml:137:106: Task exited with code 255
```

In beautiful colored image:

![image](https://user-images.githubusercontent.com/924196/43762827-4f39b654-9a29-11e8-9df2-4665df7992a0.png)

Am I doing something wrong?

Thanks for help

**Comment by vitek-rostislav** [comment]
Hi @TomasVotruba, thx for reporting the issue :+1: 

ad the docs structure - we are aware of the fact that it is not optimal, we already have a user story for fixing that. It should be done before the beta is released, I hope :slightly_smiling_face:  I have added your complaint into the user story :wink: 

ad failing standards - I believe this is caused by the fact that you did not run the proper dev build of the app (`build-demo-dev`) that contains phing targert for generating the class that the `phpstan` reported as missing - the target name is `tests-acceptance-build`

**Comment by TomasVotruba** [comment]
Hey, 

as for standards. I need to only run coding standard, not `build-demo-dev`.
Running static analysis should not depend on any hidden command.

**Comment by PetrHeinz** [comment]
Well, the command generates a PHP class. Static analysis depends on the class being there. It's essentially the same thing as not having composer dependencies installed. How would you solve it?

**Comment by TomasVotruba** [comment]
That's wrong dependency. I only want to run coding standards/static analysis. Nothing more.

I would not analyse a file that doesn't exist.

**Comment by PetrHeinz** [comment]
Are you sure? I mean, the class is used in the project - if it's missing phpstan will find a non-existing class and trigger an error.

**Comment by TomasVotruba** [comment]
I'm sure I want to run phsptan check after `composer install`.

**Comment by PetrHeinz** [comment]
Configuring the Codeception class creation to run automatically after `composer install` should be pretty easy and solve the issue imo.

**Comment by TomasVotruba** [comment]
Thank you, that would be great :+1: 

Also, this check should be in one of travis/circleci builds. It's common practise to run `composer install` and then cs/static analysis to make sure people can contribute  project with ease.

**Comment by ondrejmirtes** [comment]
I think this should be properly solved by adding `depends` with all the prerequisite tasks to the Phing `standards` target.

**Comment by PetrHeinz** [comment]
@ondrejmirtes That's a good idea! The `tests-acceptance-build` target is really quick, so execution time is not an issue, and the standards checks do in fact depend on it.

**Comment by PetrHeinz** [comment]
We've fixed the behavior in #400 by setting up a phing dependency. Thanks for the nicely reported issue :+1:
